README.md

```diff
@@ -63,8 +63,10 @@ The goal of this project is to make the easiest, fastest, and most painless way
 **Note: As Gitea is a [Gogs](https://github.com/gogits/gogs) fork, tutorials and documentation related to gogs applies to Gitea too**
 
 How to install Gitea:
+_Do note that this Gitea branch is **not** stable enough for builds_
+_Build from tags for the latest stable build, otherwise for development purposes, build from master_
```

Comment:
Hopefully, this will be a temporary line in the README and as such, I do not feel that accentuating that with bold fits.